Fix failing query tests

Test fails in CI:
```bash
=========================== short test summary info ============================
FAILED util/test/test_query.py::test_all - AssertionError: assert 0.0 > 0.0
============ 1 failed, 116 passed, 36 warnings in 329.83s (0:05:29) ============
```
Full output:

```bash
CO2_ADDR = '0xe56cc1804705e366e89f3b736c769894d90a9c35'
    @enforce_types
    def _test_stakes(CO2_ADDR: str):
        pools = query.getPools(CHAINID)
        st, fin, n = QUERY_ST, len(brownie.network.chain), 500
        rng = BlockRange(st, fin, n)
        stakes = query.getStakes(pools, rng, CHAINID)
        assert OCEAN_ADDR in stakes, stakes.keys()
        assert CO2_ADDR in stakes, (CO2_ADDR, stakes.keys())
        for basetoken_address in [OCEAN_ADDR, CO2_ADDR]:
            for stakes_at_pool in stakes[basetoken_address].values():
                assert len(stakes_at_pool) > 0
>               assert min(stakes_at_pool.values()) > 0.0
E               AssertionError: assert 0.0 > 0.0
E                +  where 0.0 = min(dict_values([2.4904214559386975, 52.617844360881215, 0.0]))
E                +    where dict_values([2.4904214559386975, 52.617844360881215, 0.0]) = <built-in method values of dict object at 0x7f6e1cbc1040>()
E                +      where <built-in method values of dict object at 0x7f6e1cbc1040> = {'0xbe5449a6a97ad46c8558a3356267ee5d2731ab5e': 2.4904214559386975, '0xe2dd09d719da89e5a3d0f2549c7e24566e947260': 52.617844360881215, '0xe53927d1e274bb39f7eba8ad9e0eee30f7504f77': 0.0}.values
util/test/test_query.py:112: AssertionError
```

--- a/util/query.py
+++ b/util/query.py
@@ -159,6 +159,9 @@ def getStakes(pools: list, rng: BlockRange, chainID: int) -> dict:
                 shares = float(d["shares"])
                 value = poolSharestoValue(shares, total_shares, base_token_liq)
 
+                if value == 0:
+                    continue
+
                 if LP_addr == SSBOT_address:
                     continue  # skip ss bot
 

